File: C/Leap_year.c
#include <stdio.h>

int main() {
     int num1;
     printf("enter the first number: ");
     scanf("%d",&num1);
    
    if(num1%4==0)
    {
        printf("these is a leap year");
    }
    else
    {
        printf("this is not a leap year");
    }

    return 0;
}

File: C#/Console/Leap_year.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Console4
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int num;
            Console.WriteLine("Enter the year: ");
            num = Convert.ToInt32(Console.ReadLine());

            if (num%4 == 0)
            {
                Console.WriteLine(num + " is a leap year");
            }
            else
            {
                Console.WriteLine("These is not a leap year");
            }
            Console.ReadLine();
        }
    }
    }

